index.html

```diff
@@ -1391,6 +1402,14 @@
           used to probe information about which <a>remote playback device</a>
           the user has without user consent.
         </p>
+        <p>
+          The user agent MAY stop <a data-lt="monitor the list of available
+          remote playback devices"> monitoring the list of available remote
+          playback devices</a> at any time if the user disables remote playback
+          (for example, through a browser setting).  In this state, pages will
+          receive no further information about the availability of <a>remote
```

Comment:
I think the normative part of the spec should then describe the behavior of _stop monitoring the list of remote playback devices_.  I'll drop the last sentence and file this as an enhancement.